submissions/Temu4/memory_pair_game/js/script.js

```diff
@@ -73,47 +67,37 @@ const rotateCard = event => {
 };
 
 const defaultCardsPosition = () => {
```

Comment:
Function must be named with verb, `setDefaultCardsPosition` will be better

Reply:
fix in this [commit](https://github.com/kottans/frontend-2019-p2p/pull/50/commits/db9452c6efae9ceb9a01a8d6f7c987afe8af3d04).